comit_node/src/btsieve/client.rs

```diff
@@ -55,10 +55,10 @@ struct QueryResponse<T> {
 impl BtsieveHttpClient {
     pub fn new(
         endpoint: &Url,
-        ethereum_poll_interval: Duration,
-        ethereum_network: &str,
         bitcoin_poll_interval: Duration,
-        bitcoin_network: &str,
```

Comment:
📣 I would suggest to type the `bitcoin_network` and `ethereum_network` with an enum to avoid this bug in the future.

Reply:
That's exactly what this PR was about ;)

Reply:
somehow I missed that indeed